.github/workflows/trigger_pact_docs_update.yml

```diff
@@ -0,0 +1,22 @@
+name: Trigger update to docs.pact.io
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - master
+    paths:
+      - '**.md'
+
+jobs:
+  run:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Trigger docs.pact.io update workflow
+        run: |
+          curl -X POST https://api.github.com/repos/pact-foundation/docs.pact.io/dispatches \
+                -H 'Accept: application/vnd.github.everest-preview+json' \
+                -H "Authorization: Bearer $GITHUB_TOKEN" \
+                -d '{"event_type": "pact-php-docs-updated"}'
+        env:
+          GITHUB_TOKEN: ${{ secrets.GHTOKENFORTRIGGERINGPACTDOCSUPDATE }}
```

Comment:
It's hard to read. If we are already using it for several projects I fine with it.

Reply:
it's set at an organisation level, but agree it is hard to read.

If we update the webhook, I'm happy to change this to be easier to grok (as we will have to update each repos workflow anyway)